src/libbson/src/bson/bson-clock.c

```diff
@@ -110,24 +113,6 @@ bson_gettimeofday (struct timeval *tv) /* OUT */
 int64_t
 bson_get_monotonic_time (void)
 {
-#if defined(BSON_HAVE_CLOCK_GETTIME) && defined(CLOCK_MONOTONIC)
-   struct timespec ts;
-   /* ts.tv_sec may be a four-byte integer on 32 bit machines, so cast to
-    * int64_t to avoid truncation. */
-   clock_gettime (CLOCK_MONOTONIC, &ts);
-   return (((int64_t) ts.tv_sec * 1000000) + (ts.tv_nsec / 1000));
-#elif defined(_WIN32)
-   /* Despite it's name, this is in milliseconds! */
-   int64_t ticks = GetTickCount64 ();
-   return (ticks * 1000);
-#elif defined(__hpux__)
-   int64_t nanosec = gethrtime ();
-   return (nanosec / 1000UL);
-#else
-#pragma message "Monotonic clock is not yet supported on your platform."
-   struct timeval tv;
-
-   bson_gettimeofday (&tv);
-   return ((int64_t) tv.tv_sec * 1000000) + tv.tv_usec;
-#endif
+   mlib_time_point now = mlib_now ();
+   return mlib_microseconds_count (now.time_since_monotonic_start);
```

Comment:
```suggestion
   return mlib_microseconds_count (mlib_now ().time_since_monotonic_start);
```
Curious: why not written as the suggested code? (Similar patterns elsewhere.)